pilot/cmd/pilot-agent/main.go

```diff
@@ -126,10 +126,6 @@ var (
 	enableGatewaySDSEnv = env.RegisterBoolVar("ENABLE_INGRESS_GATEWAY_SDS", false,
 		"Enable provisioning gateway secrets. Requires Secret read permission").Get()
 
-	// TODO: This is already present in ProxyConfig !!!
-	trustDomainEnv = env.RegisterStringVar("TRUST_DOMAIN", "",
```

Comment:
shall we remove `trust-domain` in manifest like https://github.com/istio/istio/blob/master/manifests/charts/istio-control/istio-discovery/files/injection-template.yaml#L165 as well for this clean up?

Reply:
thanks for pointing it